repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/ProjectScanner.java

```diff
@@ -270,11 +268,11 @@ private boolean testBuild(Build build, boolean targetFailing) {
                                 // testInfo can be null if the build tool is unknown
                                 if (testInfo != null && (testInfo.getFailing() > 0 || testInfo.getErrored() > 0)) {
                                     this.totalBuildInJavaFailingWithFailingTests++;
-                                    if (targetFailing) {
+                                    if (targetFailing) { // REPAIR mode
```

Comment:
Then we should completely refactor this to really use the RepairnatorConfig.LauncherMode here. It would be much more explicit about what we're doing.

Reply:
Ok. In fact I observed when doing the refactoring with `LauncherUtils` that the only module we don't use config is the scanner. It's probably a good idea to put it now. I will do the changes.

Reply:
> It's probably a good idea to put it now.

Completely!
I propose you do the changes in another PR: it will be easier to see the changes properly. Once merge you'll be able to change this one. Ok ?

Reply:
Ok.